src/style.css

```diff
@@ -0,0 +1,21 @@
+body {
+  font-family: Arial, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin-top: 0;
+}
+
+.article,
+.preamble {
```

Comment:
why you need two class for `h2`, they do the same things?

Reply:
article and preamble have class article - is good for semantic ?